### PR TITLE
chore: Normalize Setup instructions in samples READMEs

### PR DESCRIPTION
**Pull request by quartzmo**
As commented in GoogleCloudPlatform/ruby-docs-samples#679, update the `Setup` section of the samples READMEs where needed. Also, add/update `Run tests` section.

### CODE REVIEW

File: google-cloud-translate/samples/README.md [hunk 1]
```diff
@@ -17,43 +17,36 @@ Authentication is typically done through [Application Default Credentials](https
...
+1. When running on App Engine or Compute Engine, credentials are already set-up.
+However, you may need to configure your Compute Engine instance with
+[additional scopes](https://cloud.google.com/compute/docs/authentication#using).
 
-1.  You can create a [Service Account key file](https://cloud.google.com/docs/authentication#service_accounts).
-    This file can be used to authenticate to Google Cloud Platform services from
-    any environment. To use the file, set the `GOOGLE_APPLICATION_CREDENTIALS`
-    environment variable to the path to the key file, for example:
+1. You can create a [Service Account key file](https://cloud.google.com/docs/authentication#service_accounts)
+. This file can be used to authenticate to Google Cloud Platform services from
```
Comment (dazuma) [inline google-cloud-translate/samples/README.md:29]: Move this orphaned period back to the previous line.

**Review by dazuma — APPROVED** [review]
One nit, otherwise LGTM.

File: google-cloud-translate/samples/README.md (hunk 1 above)
Comment (quartzmo) [reply google-cloud-translate/samples/README.md]: Thanks, done.

**Comment by quartzmo** [comment]
Samples CI failing due to several flake tests unrelated to this PR.

**Pull request merged**